lib/envy.ex

```diff
@@ -85,7 +85,7 @@ defmodule Envy do
     end)
   end
 
-  defp blank_entry?(string) do
-    string == ""
+  defp invalid_entry?(string) do
+    string == "" || String.match?(string, ~r(^\s*#))
```

Comment:
I think it would be worth it to name this match function comment_line?(string) what do you think?